core/object.h

```diff
@@ -445,7 +445,9 @@ class Object {
 			int reference_count;
 			Connection conn;
 			List<Connection>::Element *cE;
-			Slot() { reference_count = 0; }
+			Slot() :
```

Comment:
Since we now use C++11 (well 17), we can initialize those directly upon declaration, which will be cleaner:
```
int reference_count = 0;
List<Connection>::Element *cE = nullptr;
```

Reply:
Fixed in latest commit